package.json

```diff
@@ -28,7 +28,7 @@
     "build": "npm run clean && tsc",
     "format": "prettier -w .",
     "release": "npm run build && release-it",
-    "test": ""
+    "test": "node --test -r ts-node/register ./tests/**/*.test.ts"
```

Comment:
TODO: Separate compiling tests with a different tsconfig, and then use built infra-code-blocks instead of referencing /src components.